### ISSUE TITLE
Has this project been abandoned?

### ISSUE THREAD

**Issue by PhantomWatson** [moxiecode/plupload#1605]
Is this project abandoned? There haven't been any commits for a year and a half, 14 open pull requests going back five years are being ignored, there are 155 open issues going back almost ten years, and there haven't been any posts in the [forum](https://www.plupload.com/punbb/) for the last half-year.

I don't wish to shame anybody for not working on this, and I understand that Moxiecode may not want to declare the project abandoned while it's still selling commercial licenses for it, but it might be helpful to other developers to put a message on the readme that says that active development/support shouldn't currently be expected, if that's indeed the case.

**Comment by jayarjo** [comment]
You are right issues going back years in the past should probably be closed as majority of them is irrelevant. Also Plupload does it's job as it is, so probably no new features are expected. Active development on this project has been wrapped. Forum is closed as it got flooded by spam bots. You are welcome to use StackOverflow instead. Project still accepts support requests for important bugfixes though. But as far as I know there were none in recent years. I see couple of useful PRs, but the rest can simply be closed.
